coap-gateway/bootstrapper/go.sum

```diff
@@ -0,0 +1,1418 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
pls remove go.sum file - we use only per repository so github.com/plgd-dev/cloud/go.sum

Reply:
done